Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM fedora:27
+MAINTAINER Arista Ansible Team <ansible@arista.com>
+
+ENV PS1='avd:\u% '
+
+# Install necessary packages
+RUN dnf -y install \
+    findutils \
+    git \
+    make \
+    python-pip \
+    rpm-build \
+    sudo
+
+# Create the /project directory and add it as a mountpoint
+WORKDIR /ansible_avd
+VOLUME ["/ansible_avd"]
+
+# Install python modules required by the repo
+RUN pip install --upgrade pip
+ADD requirements.txt /tmp/requirements.txt
+RUN pip install --trusted-host pypi.python.org -r /tmp/requirements.txt
```

Comment:
Seems to not be required since it is included in `requirements-dev.txt`

Reply:
@titom73 I agree.  I'll do away with this since it adds an extra unneeded step to the dockerfile.